README.md

```diff
@@ -65,7 +65,7 @@ module "power-infrastructure" {
 |------|---------|
 | <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.1.0 |
 | <a name="requirement_ibm"></a> [ibm](#requirement\_ibm) | >= 1.44.3 |
-| <a name="requirement_time"></a> [time](#requirement\_time) | 0.8.0 |
+| <a name="requirement_time"></a> [time](#requirement\_time) | >= 0.8.0 |
```

Comment:
Is that stable enough to rely on a newest (and probably not tested) dependency?

Reply:
@ocofaigh  suggested to add a range here.
Any comments ?